docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
+    volumes:
+      - ./:/app
+      - my_media:/app/files/media
+      - my_static:/app/staticfiles
```

Comment:
Ensure that the volume paths (`my_media` and `my_static`) are correctly set up in your Django settings to match these paths, so that media and static files are correctly served.